apps/emqx/test/emqx_crl_cache_SUITE.erl

```diff
@@ -941,10 +941,13 @@ t_revoked(Config) ->
         {port, 8883}
     ]),
     unlink(C),
-    ?assertMatch(
-        {error, {ssl_error, _Sock, {tls_alert, {certificate_revoked, _}}}}, emqtt:connect(C)
-    ),
-    ok.
+    case emqtt:connect(C) of
+        {error, {ssl_error, _Sock, {tls_alert, {certificate_revoked, _}}}} ->
```

Comment:
I remember this is otp25 specific behavior.

Reply:
no, it's a race condition.

Reply:
if you mean it's a race condition which happens on otp25 only? could be ;)